src/calculateRentalCost.js

```diff
@@ -20,7 +20,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let discount = 0;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
+
+  return cost * days - discount;
```

Comment:
```suggestion
  return (cost * days) - discount;
```
Добавляй скобочки для читаемости